src/utils/map/coordinates.ts

```diff
@@ -315,3 +315,12 @@ export function parseGeomFromString(input: string) {
 
   return results;
 }
+
+export function convertFeatureCollectionProjection(data: any, from: string, to: string) {
+  const features = new GeoJSON().readFeatures(_.cloneDeep(data), {
+    dataProjection: from,
+    featureProjection: to,
```

Comment:
For optimal performance, I would advise to skip using `cloneDeep` when the source and target objects are identical.

Reply:
good catch! fixed